### PR TITLE
Add i18n translation

### PR DESCRIPTION
**Pull request by PierrickGT**
Issue: https://github.com/status-im/ens-usernames/issues/76

### CODE REVIEW

**Comment by chadyj** [comment]
Nice one!
@bgits @jeluard @3esmit please take a look

**Comment by chadyj** [comment]
@PierrickGT What's your plan for the other text that hasn't yet been converted to strings?

**Comment by 3esmit** [comment]
How the language is changed? Automatically based on system configuration?

File: app/languages/index.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import english from './english.json';
```
Comment (chadyj) [inline app/languages/index.js]: For easier identification and consistency with our other projects can you please use ISO language codes, such as `en.json` and `fr.json`?

File: app/languages/index.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+import french from './french.json';
+
+const translations = {
+  en: english,
```
Comment (chadyj) [inline app/languages/index.js]: do we need this step if we name the files according to ISO codes?

File: app/store/configureStore.js [hunk 3]
```diff
@@ -1,11 +1,13 @@
-import { createStore, applyMiddleware } from 'redux';
+import { applyMiddleware, compose, createStore } from 'redux';
 import rootReducer from '../reducers/rootReducer';
 import thunk from 'redux-thunk';
 
 const store = createStore(
   rootReducer,
-  window.__REDUX_DEVTOOLS_EXTENSION__ && window.__REDUX_DEVTOOLS_EXTENSION__(),
```
[the remark this answers is not in the document]
Comment (PierrickGT) [reply app/store/configureStore.js]: Yes, I couldn't start the app in dev mode without it.

**Pull request merged**